app/Vendor/adldap/adldap/src/Connections/Ldap.php

```diff
@@ -337,7 +337,7 @@ public function connect($hostname, $port = '389')
     {
         $protocol = $this::PROTOCOL;
 
-        if ($this->isUsingSSL()) {
+        if ($this->isUsingSSL() || $this->isUsingTLS()) {
```

Comment:
The `$this::PROTOCOL_SSL` (`'ldaps://'`) URI is only used for SSL encrypted connections, not connections with STARTTLS

Reply:
```suggestion
        if ($this->isUsingSSL()) {
```

Of course... I just saw that using port 636 and ldaps:// is deprecated.

PS: With the next major release, the [Adldap](https://github.com/adldap/adLDAP) library will be removed and only [FreeDSx\Ldap](https://github.com/FreeDSx/LDAP) will be used.